extensions/notebook/src/extension.ts

```diff
@@ -109,6 +109,8 @@ export async function activate(extensionContext: vscode.ExtensionContext): Promi
 		await vscode.commands.executeCommand('vscode.open', vscode.Uri.parse(urlToOpen));
 	}));
 
+	extensionContext.subscriptions.push(vscode.commands.registerCommand('notebook.command.revealInBooksViewlet', (uri: vscode.Uri, shouldReveal: boolean) => bookTreeViewProvider.revealActiveDocumentInViewlet(uri, shouldReveal)));
```

Comment:
So we don't need this for untitled view-let?

Reply:
I chatted about this and thought about this some more, and we actually don't. Given the latest mockups it doesn't include the untitled viewlet either, so let's leave this alone for now.

Reply:
Do I need change the way we're opening the TSG books then? Because currently they open in the untitled view-let, probably will fail with the current changes.